tests/zolar/bank/pay.test.js

```diff
@@ -0,0 +1,71 @@
+const { getAddressFromPrivateKey } = require("@zilliqa-js/zilliqa")
+const { default: BigNumber } = require("bignumber.js");
+const { ZERO_ADDRESS, getPrivateKey, deployHuny, deployGuildBank } = require('../../../scripts/zolar/bank/deploy.js');
+const {callContract} = require('../../../scripts/call.js')
+
+let privateKey, address, hunyContract, bankContract
+
+beforeAll(async () => {
+  privateKey = getPrivateKey();
+  address = getAddressFromPrivateKey(privateKey).toLowerCase();
+  hunyContract = await deployHuny()
+  bankContract = await deployGuildBank({ hiveAddress: ZERO_ADDRESS, hunyAddress: hunyContract.address })
+})
+
+test('add minter', async () => {
+  const txAddMinter = await callContract(privateKey, hunyContract, "AddMinter", [{
+    vname: 'minter',
+    type: 'ByStr20',
+    value: address,
+  }], 0, false, false);
+  
+  const state = await hunyContract.getState()
+
+  expect(txAddMinter.status).toEqual(2)
+  expect(txAddMinter.receipt.success).toEqual(true)
+  expect(state.minters).toHaveProperty(address)
+})
+
+test('mint huny', async () => {
+  const txMint = await callContract(privateKey, hunyContract, "Mint", [{
+    vname: 'recipient',
+    type: 'ByStr20',
+    value: address,
+  }, {
+    vname: 'amount',
+    type: 'Uint128',
+    value: new BigNumber(1).shiftedBy(12 + 3),
+  }], 0, false, false)
+  
+
+  expect(txMint.status).toEqual(2)
+  expect(txMint.receipt.success).toEqual(true)
+  // KIV assertion to check balance before and after tx
```

Comment:
Minting should not be tested under guild bank, and there is no need to test a ZRC2 function for the purpose of this this repo.